### DISCUSSION TITLE
hipaa [Jun 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2299] [Discord Threads]
> 🙋‍♂️ **jaan**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### we've not looked into it  
#  
> 💁 **o1lab**  
#### you've ideas on what the process looks like  
#  
> 🙋‍♂️ **jota**  
#### Im also within this use case (using NocoDB to integrate “stubborn data”: data that I’ve lost hope I’ll be able to import automatically and NocoDB is a good solution for our finance team to input really sensitive data so we can connect to our dashboards with minimal manual work  
#  
> 🙋‍♂️ **jota**  
#### And I was also afraid for trusting sensitive data. But shouldnt the fact of being self hosted + databse on our infrastructure to be almost enough?  
#  
> 🙋‍♂️ **jota**  
#### The only missing option would be some backdoor sending data somewhere else, but we’ll put our nocodb behind a VPN without internet access  
#  
> 💁 **o1lab**  
#### hey Jota  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979095108857253909)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
